crypt-gpgme.c

```diff
@@ -127,6 +127,24 @@ static char *current_sender = NULL;
  * General helper functions.
  */
 
+/* accomodate for a redraw if needed */
+static void
+redraw_if_needed (gpgme_ctx_t ctx)
+{
+#if GPGME_VERSION_NUMBER < 0x010801
+  /* The "redraw" flag is not available in gpgme < 1.8.1. In this case, stay on
+   * the safe side and always redraw. */
+  (void)ctx;
+  mutt_need_hard_redraw ();
+#else
+  const char *s = gpgme_get_ctx_flag (ctx, "redraw");
```

Comment:
Documentation of this function states that it returns NULL if "name" (second argument) is unknown, so it seems the #if block is not necessary